index.html

```diff
@@ -819,20 +819,26 @@
 		</section>
 		</div>
 		<section>
+			<h2 id="tracking-legacy-mouse-pos">Tracking the effective position of legacy mouse pointer</h2>
+			<div>
+			<p>While only <a href="#the-primary-pointer">primary pointers</a> can produce compatibility mouse events, <a href="#multiple-primary-pointers">multiple primary pointers</a> can be active simultaneously, each producing its own stream of compatibility mouse events.  Since all these compatibility events would appear to a legacy mouse-specific code to be coming from a single mouse hardware, user agents are encouraged to guarantee that the compatibility mouse events are consistent from a single device perspective.  For mouse transition events (i.e., <code>mouseover</code>, <code>mouseout</code>, <code>mouseenter</code> and <code>mouseleave</code>), this means the entry/exit state for every event target is valid as implied by [<cite><a class="bibref" href="#bib-DOM-LEVEL-3-EVENTS">DOM-LEVEL-3-EVENTS</a></cite>].  Users agents MAY guarantee this by maintaining the <dfn id="dfn-legacy-mouse-pos">effective position of legacy mouse pointer</dfn> in the document as follows.</p>
+			<p>Right before firing a <code>pointerdown</code>, <code>pointerup</code> or <code>pointermove</code> event, the user agent MAY run the following steps:</p>
```

Comment:
The way to officially reference another spec normatively is [[!DOM-LEVEL-3-EVENTS]] - see the [ReSpec guide](https://www.w3.org/respec/guide.html#references).

However "DOM-LEVEL-3" is the old name for that spec and so kinda confusing now.  Would you mind updating all the various references to it to "UIEVENTS" instead?  Probably best to do that in a separate CL to avoid polluting this one if you don't mind.